utils/rth

```diff
@@ -233,7 +239,7 @@ def main():
                                      args.t, args.S, args.s, args.lib_prefix,
                                      args.lib_suffix,
                                      args.additional_compile_flags,
-                                     args.no_backward_deployment,
+                                     args.backward_deployment,
```

Comment:
This seems like a step backwards. Shouldn't tests be testing backwards deployment by default?

Reply:
The flag means "test backward deployment only". Ie, normally we test all possible values for the (app, library) matrix, (before, before), (before, after), (after, before), (after, after). With this flag we only test (after, before) and (after, after), since the app doesn't *build* with the "before" library (it references new declarations).

Reply:
Can you rename it to `backward-deployment-only`, then?